### DISCUSSION TITLE
(🎁) Include `.idea`/`.vscode` config directories

### DISCUSSION THREAD

**Discussion by KotlinIsland** [johnthagen/python-blueprint#220] [Ideas]

**Comment by johnthagen** [comment]
Hey @KotlinIsland good to see you over here. I've enjoyed your other work in the community.

A few initial thoughts

- I don't personally use VS Code for development, so I would not be comfortable keeping that up to date as it's important to me that anything in python-blueprint I personally vet and keep updated
- For Pycharm, I've had mixed success in the past trying to save portions of the `.idea` folder in a way that it works well across a team, since most of what is in there is specific to the user, but some can be shared. For example suggesting the Ruff plugin would be potentially nice
  - I will admit that I haven't looked into this deeply lately, so if there are some guides from JetBrains on that, I would be interested in taking a read

**Comment by lovepamthailand1672538** [comment]
bc1qnppvarg8f0gzcfhmytuky0l5ncc90e5kyuxl8q
